Library/Homebrew/dev-cmd/update-python-resources.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def update_python_resources_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update-python-resources` [<options>] <formula>
+
+        Update versions for PyPI resource blocks in <formula>.
+      EOS
+      switch "-p", "--print-only",
+             description: "Print the updated resource blocks instead of changing the formula."
+      named :formula
+    end
+  end
+
+  def update_python_resources
+    update_python_resources_args.parse
+
+    formula = args.formulae.first
+
+    pypi_resources = formula.resources.select do |resource|
+      resource.url.match?(%r{^https?://files.pythonhosted.org/packages/})
```

Comment:
The `?` isn't needed, so using `resource.url.start_with?("https://files.pythonhosted.org/packages/")` would be more performant.

Reply:
Thanks 👍